Cabal-syntax/src/Distribution/Types/Benchmark.hs

```diff
@@ -47,17 +47,11 @@ instance Monoid Benchmark where
 
 instance Semigroup Benchmark where
     a <> b = Benchmark {
-        benchmarkName      = combine' benchmarkName,
+        benchmarkName      = combineName a b benchmarkName "benchmark",
```

Comment:
These (changes in `Type/`) are just a fix of the annoying `Semigroup` target instances (which crashed when the names of the targer were the same).